app/views/downloads/_download_preface.html.erb

```diff
@@ -66,14 +66,20 @@
       <ul>
         <% related_works.each do |work| %>
           <li>
-            <% relation_text = work.translation ?
-              "A translation of %{work_link} by %{author_link}" :
-              "Inspired by %{work_link} by %{author_link}"
-            %>
+            <%  work_info = if work.parent.unrevealed?
+                              "a work in an unrevealed collection"
+                            else
+                              "%{work_link} by %{author_link}"
+                            end
+                relation_text = if work.translation
+                                  "A translation of %{work_info}."
+                                else
+                                  "Inspired by %{work_info}."
+                                end %>
             <% work_link = link_to work.parent.title.html_safe, url_for(work.parent) %>
             <% author_link = byline(work.parent, visibility: 'public', only_path: false) %>
 
-            <%= ts(relation_text, work_link: work_link, author_link: author_link).html_safe %>
+            <%= ts(relation_text, work_link: work_link, author_link: author_link, work_info: work_info).html_safe %>
```

Comment:
Does this work when the parent isn't unrevealed? I would have thought I18n would stop after the first substitution.

Reply:
I've spoke with Translation, and if it turns out it doesn't work, it would actually be better to have this i18ned like:

```
<% relation = work.translation ? "translation_of" : "inspired_by" %>

<% if work.parent.unrevealed? %>
  <%= t(".#{relation}.unrevealed") %>
<% else %>
  <%= t(".#{relation}.revealed", work_link: work_link, creator_link: author_link) %>
<% end %>
```

Giving Translation this:
```
  inspired_by:
    revealed: Inspired by %{work_link} by %{creator_link}.
    unrevealed: Inspired by a work in an unrevealed collection.
  translation_of:
    revealed: Translation of %{work_link} by %{creator_link}.
    unrevealed: Translation of a work in an unrevealed collection.
```